### DISCUSSION TITLE
关于边的问题

### DISCUSSION THREAD

**Discussion by qianqian05723** [didi/LogicFlow#1413] [General]
这个边可以只有起始点，没有结束点嘛？类似这样
![image](https://github.com/didi/LogicFlow/assets/17306867/f222fa8c-5914-4125-916a-e9a5c0f3bacd)

**Comment by boyongjiong** [comment]
目前的设计是不可以的，边的路径计算基于起始节点信息。

后面我们可能会重构一下增加这个能力，但节奏还没定好
